openforcefield/utils/toolkits.py

```diff
@@ -2902,11 +2902,16 @@ def register_toolkit(self,
 
         # Raise exception if not available.
         if not toolkit_wrapper.is_available():
-            msg = "Unable to load toolkit {}.".format(toolkit_wrapper)
+            msg = "Unable to load toolkit '{}'. ".format(toolkit_wrapper._toolkit_name)
             if exception_if_unavailable:
                 raise ToolkitUnavailableException(msg)
             else:
-                logger.warning(msg)
+                if 'OpenEye' in msg:
+                    msg += "Versions of the Open Force Field Toolkit past 0.2.0 (April 2019) do not require the " \
```

Comment:
Suggest alternative:
```
The Open Force Field Toolkit does not require the OpenEye Toolkits, and can use RDKit/AmberTools instead.
However, if you have a valid license for the OpenEye Toolkits, consider installing them for faster performance 
and additional file format support: https://docs.eyesopen.com/toolkits/python/quickstart-python/linuxosx.html
OpenEye offers free Toolkit licenses for academics: https://www.eyesopen.com/academic-licensing
```

Reply:
I like this message better. Added in 498cfff